public/resources/stylesheets/main.css

```diff
@@ -824,6 +831,96 @@ h1, h2, h3, h4, h5, h6 {
   cursor: pointer;
 }
 
+.adminMenu {
+  color: inherit;
+  display: grid;
+}
+.adminMenu-modal {
+  background-color: #FCF6F4;
+  border: 1px solid #ccc;
+  margin-top: 20px;
+  width: 80%;
+}
+.adminMenu-showpost {
+  text-align: left;
+  width: 100%;
+}
+.adminMenu-show {
+  padding: 20px;
+  border-radius: 10px;
+  max-width: 90%;
+  box-shadow: 0 0 5px rgb(177, 165, 165);
+  color: #0c0a0a;
+  margin: 20px auto;
+  animation: aparecer 1s ease forwards;
+}
+.adminMenu-header {
+  align-items: center;
+  display: flex;
+  padding: 10px 0;
+}
+.adminMenu-header img {
+  border-radius: 50%;
+  height: 50px;
+  width: 50px;
+}
+.adminMenu-header h4 {
+  font-size: 2rem;
+  text-align: left;
+}
+.adminMenu-header p {
+  font-size: 1.5rem;
+  text-align: left;
+}
+.adminMenu-list {
+  display: grid;
+  gap: 1rem;
+  list-style: none;
+  max-height: 18.75rem;
+  overflow: auto;
+}
+.adminMenu-list-item {
+  cursor: pointer;
+}
+.adminMenu-list-item:hover {
+  background-color: #BEF4FE;
+  border-radius: 5px;
+}
+.adminMenu-list-item.spinner-container {
+  display: flex;
+  justify-content: center;
+  padding: 1rem;
+}
+.adminMenu-list-item.spinner-container .spinner {
+  height: 4rem;
+  width: 4rem;
+}
+.adminMenu-list-item a {
+  font-size: 20px;
+  padding: 4px 0;
+}
+.adminMenu-tabs {
+  border-bottom: 1px solid #bfbfbf;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  list-style: none;
+  margin-bottom: 1.25em;
+}
+.adminMenu-tabs-item {
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  border: 1px solid transparent;
+  cursor: pointer;
+  padding: 4px 0;
+}
+.adminMenu-tabs-item:hover:not(.active) {
+  background-color: #faefec;
+}
+.adminMenu-tabs-item.active {
+  border-color: #bfbfbf #bfbfbf #FCF6F4;
```

Comment:
for this project, we declare variables on the `public/resources/stylesheets/abstracts/_variables.scss` file, in this way we ensure do not reapeat or make a mess with the colors if are need to be used in some specific context